src/eth.rs

```diff
@@ -554,6 +572,38 @@ impl Provider {
         self.send_request_and_parse_response::<Bytes>(action)
     }
 
+    /// Gets an entry from the KiMap.
+    ///
+    /// # Parameters
+    /// - `entryhash`: The entry to get from the KiMap.
+    /// # Returns
+    /// A `Result<(Address, Address, Option<Bytes>), EthError>` representing the TBA, owner, and value if the entry is a note.
+    pub fn get(&self, entryhash: &str) -> Result<(Address, Address, Option<Bytes>), EthError> {
+        let get_call = getCall {
+            entryhash: FixedBytes::<32>::from_str(entryhash)
+                .map_err(|_| EthError::InvalidParams)?,
+        }
+        .abi_encode();
+
+        let tx_req = TransactionRequest::default()
+            .input(TransactionInput::new(get_call.into()))
+            .to(Address::from_str(KIMAP).unwrap());
```

Comment:
Yeah, vastly better if we get an Address from a struct and have `get()` and whatnot as fns of that struct impl.